napari_allencell_annotator/model/annotation_model.py

```diff
@@ -133,7 +135,7 @@ def get_annotations(self) -> dict[Path, list[Any]]:
         return self._created_annotations
 
     def add_annotation(self, file_path: Path, annotation: list[Any]):
-        self._created_annotations[file_path] = annotation
+        self._created_annotations[file_path]: list[Any] = annotation
```

Comment:
this is already typed and doesnt need it here

Reply:
removed in https://github.com/aics-int/napari-allencell-annotator/pull/60/commits/251e3c9fef46a8a1794311e663c5fad548acf54e